core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -117,6 +117,9 @@ constructor(
 
     private val _surfaceRequest = MutableStateFlow<SurfaceRequest?>(null)
 
+    private val cameraSessionZoomScale = MutableStateFlow(1f)
+    private var prevCameraSessionLensFacing: LensFacing? = null
```

Comment:
These don't need to be instance variables. They should be moved to be local variables right after line 291.